rtl/include/eclhelper_base.hpp

```diff
@@ -1371,7 +1371,7 @@ class ECLRTL_API CLibraryMemorySpillSplitArg : public CThorSpillArg
 {
 public:
     CLibraryMemorySpillSplitArg(unsigned _tempUsageCount, const char * _filename, IOutputMetaData * _meta) :
-        meta(_meta), filename(_filename), tempUsageCount(_tempUsageCount) {}
+        meta(_meta), filename(_filename), tempUsageCount(_tempUsageCount)  {}
```

Comment:
pedantic: gained a space.

Reply:
I'm stockpiling in case spaces are in short supply after Brexit